### PR TITLE
Dispatch change event for file explorer

### PR DESCRIPTION
**Pull request by aliabid94**
Fixes: #7432

### CODE REVIEW

File: js/fileexplorer/Index.svelte [hunk 1]
```diff
@@ -37,6 +38,11 @@
 	export let interactive: boolean;
 
 	$: rerender_key = [root_dir, glob, ignore_glob];
+
+	$: if (JSON.stringify(value) !== JSON.stringify(old_value)) {
```
Comment (freddyaboulton) [inline js/fileexplorer/Index.svelte:42]: Let's add a unit test for this - otherwise looks good to me @aliabid94 !

**Review by abidlabs — APPROVED** [review]
Agree with @freddyaboulton!

**Review by dawoodkhan82 — APPROVED** [review]

File: js/fileexplorer/Index.svelte (hunk 1 above)
Comment (aliabid94) [reply js/fileexplorer/Index.svelte:42]: done

**Review by pngwn — APPROVED** [review]
Looks good. Just updated the branch to see if the tests pass, they are finae locally so not sure what the issue is. Thanks @aliabid94!

**Pull request merged**